react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from '../Square';
+
+import style from './styles.scss';
+
+class Board extends Component {
+  renderSquare = i => <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
Impor the required elements directly from react (just like you did with Component)
```suggestion
      <Fragment>
```